charts/sftpgo/values.yaml

```diff
@@ -56,6 +56,15 @@ envFrom: []
 # See the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables) for details.
 env: {}
 
+# -- Additional environment variables values of which are populated from specific [secrets and keys](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables)
```

Comment:
Let's link to [this](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables) page instead like this:

```
# See the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables) for details.
```

I'd also add a comment to `env` saying that it's a simplified syntax to allow key-value pairs and that `envVars` allows a more detailed syntax.

Reply:
done